diff --git a/node_modules/gojs/release/go-module.js b/node_modules/gojs/release/go-module.js
index 03d8e7a..181a14c 100644
--- a/node_modules/gojs/release/go-module.js
+++ b/node_modules/gojs/release/go-module.js
@@ -803,7 +803,10 @@ Q.prototype.getInputOption=Q.prototype.yl;Q.prototype.resetRenderingHints=Q.prot
 Q.prototype.findObjectsIn=Q.prototype.sf;Q.prototype.findPartsIn=Q.prototype.Vw;Q.prototype.findObjectsAt=Q.prototype.di;Q.prototype.findPartsAt=Q.prototype.Uy;Q.prototype.findObjectAt=Q.prototype.ec;Q.prototype.findPartAt=Q.prototype.wl;Q.prototype.focusObject=Q.prototype.gz;Q.prototype.alignDocument=Q.prototype.oy;Q.prototype.zoomToRect=Q.prototype.kA;Q.prototype.zoomToFit=Q.prototype.zoomToFit;Q.prototype.diagramScroll=Q.prototype.Mw;Q.prototype.focus=Q.prototype.focus;
 var li=new pb,Di=null,Og=void 0!==x.document,yi=null,zi="",rh=new D(Q,"None",0),Yi=new D(Q,"Uniform",1),Zi=new D(Q,"UniformToFill",2),Ef=new D(Q,"CycleAll",10),If=new D(Q,"CycleNotDirected",11),Kf=new D(Q,"CycleNotDirectedFast",12),Lf=new D(Q,"CycleNotUndirected",13),Ff=new D(Q,"CycleDestinationTree",14),Hf=new D(Q,"CycleSourceTree",15),Rh=new D(Q,"DocumentScroll",1),Th=new D(Q,"InfiniteScroll",2),vi=new D(Q,"TreeParentCollapsed",21),Dk=new D(Q,"AllParentsCollapsed",22),Ek=new D(Q,"AnyParentsCollapsed",
 23),Fk=new E,Gk="2.2.14",Ak="image",Hk=null,ji=!1;
-function ki(){if(Og){var a=x.document.createElement("canvas"),b=a.getContext("2d"),c=Ma("7ca11abfd022028846");b[c]=Ma("398c3597c01238");for(var d=["5da73c80a36455d6038e4972187c3cae51fd22",ra.Dx+"4ae6247590da4bb21c324ba3a84e385776",md.xF+"fb236cdfda5de14c134ba1a95a2d4c7cc6f93c1387",J.za],e=1;5>e;e++)b[Ma("7ca11abfd7330390")](Ma(d[e-1]),10,15*e);b[c]=Ma("39f046ebb36e4b");for(c=1;5>c;c++)b[Ma("7ca11abfd7330390")](Ma(d[c-1]),10,15*c);if(4!==d.length||"5"!==d[0][0]||"7"!==d[3][0])md.prototype.Ld=md.prototype.Fw;
+  function ki() {
+    if (Og) {
+      var a = x.document.createElement("canvas"), b = a.getContext("2d");
+// ,c=Ma("7ca11abfd022028846");b[c]=Ma("398c3597c01238");for(var d=["5da73c80a36455d6038e4972187c3cae51fd22",ra.Dx+"4ae6247590da4bb21c324ba3a84e385776",md.xF+"fb236cdfda5de14c134ba1a95a2d4c7cc6f93c1387",J.za],e=1;5>e;e++)b[Ma("7ca11abfd7330390")](Ma(d[e-1]),10,15*e);b[c]=Ma("39f046ebb36e4b");for(c=1;5>c;c++)b[Ma("7ca11abfd7330390")](Ma(d[c-1]),10,15*c);if(4!==d.length||"5"!==d[0][0]||"7"!==d[3][0])md.prototype.Ld=md.prototype.Fw;
 Hk=a}}Q.className="Diagram";Q.fromDiv=function(a){var b=a;"string"===typeof a&&(b=x.document.getElementById(a));return b instanceof HTMLDivElement&&b.H instanceof Q?b.H:null};Q.inherit=function(a,b){function c(){}if(Object.getPrototypeOf(a).prototype)throw Error("Used go.Diagram.inherit defining already defined class \n"+a);c.prototype=b.prototype;a.prototype=new c;a.prototype.constructor=a};Q.useDOM=function(a){Og=a?void 0!==x.document:!1};Q.isUsingDOM=function(){return Og};Q.None=rh;Q.Uniform=Yi;
 Q.UniformToFill=Zi;Q.CycleAll=Ef;Q.CycleNotDirected=If;Q.CycleNotDirectedFast=Kf;Q.CycleNotUndirected=Lf;Q.CycleDestinationTree=Ff;Q.CycleSourceTree=Hf;Q.DocumentScroll=Rh;Q.InfiniteScroll=Th;Q.TreeParentCollapsed=vi;Q.AllParentsCollapsed=Dk;Q.AnyParentsCollapsed=Ek;function ri(){this.ay=null;this.j="zz@orderNum";"63ad05bbe23a1786468a4c741b6d2"===this._tk?this.pd=this.j=!0:this.pd=null}
 function sj(a,b){b.Zb.setTransform(b.kc,0,0,b.kc,0,0);b.Zb.Oc();if(null===a.pd)a:{b="f";var c=x[Ma("76a715b2f73f148a")][Ma("72ba13b5")];a.pd=!0;if(Og){var d=Q[Ma("76a115b6ed251eaf4692")];if(d)for(var e=Fk.iterator;e.next();){d=e.value;d=Ma(d).split(Ma("39e9"));if(6>d.length)break;var f=Ma(d[1]).split(".");if("7da71ca0"!==d[4])break;var g=Ma(ra[Ma("6cae19")]).split(".");if(f[0]>g[0]||f[0]===g[0]&&f[1]>=g[1]){var h=x[Ma("74a900bae4370f8b51")][Ma("6fbb13a1c2311e8a57")].toLowerCase();g=h[Ma("73a612b6fb191d")](Ma("3aad1ab6e022098b4dc4"));
diff --git a/node_modules/gojs/release/go.js b/node_modules/gojs/release/go.js
index 234b37d..77e1011 100644
--- a/node_modules/gojs/release/go.js
+++ b/node_modules/gojs/release/go.js
@@ -803,7 +803,22 @@ Q.prototype.getInputOption=Q.prototype.yl;Q.prototype.resetRenderingHints=Q.prot
 Q.prototype.findObjectsIn=Q.prototype.sf;Q.prototype.findPartsIn=Q.prototype.Vw;Q.prototype.findObjectsAt=Q.prototype.di;Q.prototype.findPartsAt=Q.prototype.Uy;Q.prototype.findObjectAt=Q.prototype.ec;Q.prototype.findPartAt=Q.prototype.wl;Q.prototype.focusObject=Q.prototype.gz;Q.prototype.alignDocument=Q.prototype.oy;Q.prototype.zoomToRect=Q.prototype.kA;Q.prototype.zoomToFit=Q.prototype.zoomToFit;Q.prototype.diagramScroll=Q.prototype.Mw;Q.prototype.focus=Q.prototype.focus;
 var li=new pb,Di=null,Og=void 0!==x.document,yi=null,zi="",rh=new D(Q,"None",0),Yi=new D(Q,"Uniform",1),Zi=new D(Q,"UniformToFill",2),Ef=new D(Q,"CycleAll",10),If=new D(Q,"CycleNotDirected",11),Kf=new D(Q,"CycleNotDirectedFast",12),Lf=new D(Q,"CycleNotUndirected",13),Ff=new D(Q,"CycleDestinationTree",14),Hf=new D(Q,"CycleSourceTree",15),Rh=new D(Q,"DocumentScroll",1),Th=new D(Q,"InfiniteScroll",2),vi=new D(Q,"TreeParentCollapsed",21),Dk=new D(Q,"AllParentsCollapsed",22),Ek=new D(Q,"AnyParentsCollapsed",
 23),Fk=new E,Gk="2.2.14",Ak="image",Hk=null,ji=!1;
-function ki(){if(Og){var a=x.document.createElement("canvas"),b=a.getContext("2d"),c=Ma("7ca11abfd022028846");b[c]=Ma("398c3597c01238");for(var d=["5da73c80a36455d6038e4972187c3cae51fd22",ra.Dx+"4ae6247590da4bb21c324ba3a84e385776",md.xF+"fb236cdfda5de14c134ba1a95a2d4c7cc6f93c1387",J.za],e=1;5>e;e++)b[Ma("7ca11abfd7330390")](Ma(d[e-1]),10,15*e);b[c]=Ma("39f046ebb36e4b");for(c=1;5>c;c++)b[Ma("7ca11abfd7330390")](Ma(d[c-1]),10,15*c);if(4!==d.length||"5"!==d[0][0]||"7"!==d[3][0])md.prototype.Ld=md.prototype.Fw;
+  function ki() {
+    if (Og) {
+      var a = x.document.createElement("canvas"), b = a.getContext("2d");
+      //  c = Ma("7ca11abfd022028846");
+      // b[c] = Ma("398c3597c01238");
+      // for (var d = ["5da73c80a36455d6038e4972187c3cae51fd22", ra.Dx + "4ae6247590da4bb21c324ba3a84e385776", md.xF + "fb236cdfda5de14c134ba1a95a2d4c7cc6f93c1387", J.za], e = 1; 5 > e; e++){
+      //   b[Ma("7ca11abfd7330390")](Ma(d[e - 1]), 10, 15 * e);
+      //   // console.log(Ma(d[e - 1]))
+      //   // Ma(d[e - 1])
+      // }
+      // b[c] = Ma("39f046ebb36e4b"); for (c = 1; 5 > c; c++){
+      //   b[Ma("7ca11abfd7330390")](Ma(d[c - 1]), 10, 15 * c);
+      //   // Ma(d[c - 1])
+      //   // console.log(Ma(d[c - 1]))
+      // }
+      //  if (4 !== d.length || "5" !== d[0][0] || "7" !== d[3][0]) md.prototype.Ld = md.prototype.Fw;
 Hk=a}}Q.className="Diagram";Q.fromDiv=function(a){var b=a;"string"===typeof a&&(b=x.document.getElementById(a));return b instanceof HTMLDivElement&&b.H instanceof Q?b.H:null};Q.inherit=function(a,b){function c(){}if(Object.getPrototypeOf(a).prototype)throw Error("Used go.Diagram.inherit defining already defined class \n"+a);c.prototype=b.prototype;a.prototype=new c;a.prototype.constructor=a};Q.useDOM=function(a){Og=a?void 0!==x.document:!1};Q.isUsingDOM=function(){return Og};Q.None=rh;Q.Uniform=Yi;
 Q.UniformToFill=Zi;Q.CycleAll=Ef;Q.CycleNotDirected=If;Q.CycleNotDirectedFast=Kf;Q.CycleNotUndirected=Lf;Q.CycleDestinationTree=Ff;Q.CycleSourceTree=Hf;Q.DocumentScroll=Rh;Q.InfiniteScroll=Th;Q.TreeParentCollapsed=vi;Q.AllParentsCollapsed=Dk;Q.AnyParentsCollapsed=Ek;function ri(){this.ay=null;this.j="zz@orderNum";"63ad05bbe23a1786468a4c741b6d2"===this._tk?this.pd=this.j=!0:this.pd=null}
 function sj(a,b){b.Zb.setTransform(b.kc,0,0,b.kc,0,0);b.Zb.Oc();if(null===a.pd)a:{b="f";var c=x[Ma("76a715b2f73f148a")][Ma("72ba13b5")];a.pd=!0;if(Og){var d=Q[Ma("76a115b6ed251eaf4692")];if(d)for(var e=Fk.iterator;e.next();){d=e.value;d=Ma(d).split(Ma("39e9"));if(6>d.length)break;var f=Ma(d[1]).split(".");if("7da71ca0"!==d[4])break;var g=Ma(ra[Ma("6cae19")]).split(".");if(f[0]>g[0]||f[0]===g[0]&&f[1]>=g[1]){var h=x[Ma("74a900bae4370f8b51")][Ma("6fbb13a1c2311e8a57")].toLowerCase();g=h[Ma("73a612b6fb191d")](Ma("3aad1ab6e022098b4dc4"));
@@ -2233,4 +2248,3 @@ CircularLayout:Gr,CircularNetwork:Xr,CircularVertex:ks,CircularEdge:ls,ForceDire
 Q.prototype.go=Ku;x.go=Ku; 'undefined'!==typeof module&&'object'===typeof module.exports&&(module.exports='undefined'!==typeof global?global.go:self.go); })();
 
   "function"===typeof define&&define.amd&&define("undefined"!==typeof global?global.go:self.go);
-  
\ No newline at end of file
